Formula/helix.rb

```diff
@@ -37,6 +37,6 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{bin}/hx -V")
-    assert_match "post-modern text editor", shell_output("#{bin}/hx -h")
+    assert_match "✔", shell_output("#{bin}/hx --health")
```

Comment:
```suggestion
    assert_match "✓", shell_output("#{bin}/hx --health")
```

The checkmark was changed in 22.08 so it would align consistently across fonts/terminals in the health table - see https://github.com/helix-editor/helix/pull/3505. (I don't think we'll change it again so it should be ok to test on 👍)

Reply:
ah thanks, unicode is hard 😝